File: 1.Intoduction to Java/From Python to Java/Exercise3.java
import java.util.Scanner;
//import java.lang.*;
class Exercise3 {
	public static void main (String args[]){
		Scanner scanner = new Scanner(System.in);
		System.out.print ("Enter the first side of the triangle:");
		int a = scanner.nextInt();
		System.out.print ("Enter the second side of the triangle:");
		int b = scanner.nextInt();
		System.out.print ("Enter the third side of the triangle:");
		int c = scanner.nextInt();
		double d = (1/4) * Math.sqrt((a+b+c)*(b-a+c)*(a-b+c)*(a+b-c));
		System.out.println (d);

	}
}

File: Preparatory weeks/PYTHON Exercises/exercise3.py
import math
a = input ('Enter the first side of the triangle: ')
b = input ('Enter the second side of the triangle: ')
c = input ('Enter the third side of the triangle: ')
a = int(a)
b = int(b)
c = int(c)
d = (1/4) * math.sqrt((a+b+c)*(b-a+c)*(a-b+c)*(a+b-c))
print (d)
